data/mock_data/generate-mock-files.sh

```diff
@@ -1,16 +1,16 @@
 set -uex
 
-touch data/mock_data/AA01_R1.fastq.gz
-touch data/mock_data/AA01_R2.fastq.gz
-touch data/mock_data/AB02_R1.fastq.gz
-touch data/mock_data/AB02_R2.fastq.gz
-touch data/mock_data/AC03_R1.fastq.gz
-touch data/mock_data/AC03_R2.fastq.gz
-touch data/mock_data/AD04_R1.fastq.gz
-touch data/mock_data/AD05_R2.fastq.gz
-touch data/mock_data/AE06_R1.fastq.gz
-touch data/mock_data/AE06_R2.fastq.gz
-touch data/mock_data/AD07_R1.fastq.gz
-touch data/mock_data/AD07_R2.fastq.gz
-touch data/mock_data/reference.fasta
-touch data/mock_data/reference.gbk
+touch ./data/mock_data/AA01_R1.fastq.gz
+touch ./data/mock_data/AA01_R2.fastq.gz
+touch ./data/mock_data/AB02_R1.fastq.gz
+touch ./data/mock_data/AB02_R2.fastq.gz
+touch ./data/mock_data/AC03_R1.fastq.gz
+touch ./data/mock_data/AC03_R2.fastq.gz
+touch ./data/mock_data/AD04_R1.fastq.gz
+touch ./data/mock_data/AD05_R2.fastq.gz
+touch ./data/mock_data/AE06_R1.fastq.gz
+touch ./data/mock_data/AE06_R2.fastq.gz
+touch ./data/mock_data/AD07_R1.fastq.gz
+touch ./data/mock_data/AD07_R2.fastq.gz
+touch ./data/mock_data/GenomeAnalysisTK.jar
+touch ./data/mock_data/NC000124.fasta
```

Comment:
@Mxrcon , just a small comment here, let's use the MTB reference fasta ID as default :)

Reply:
Yes sure!